go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/target/goalert
 
-go 1.22.0
+go 1.22
```

Comment:
the `.0` was causing my intellisense to stop working

Reply:
IIRC adding it was to workaround a separate bug that caused it to break